fedmsg_meta_fedora_infrastructure/fedimg.py

```diff
@@ -35,31 +34,31 @@ def subtitle(self, msg, **config):
             if msg['msg']['status'] == "started":
                 name = msg['msg']['image_name']
                 dest = msg['msg']['destination']
-                tmpl = self._('Image {image_name} started uploading to {dest}')
```

Comment:
Yeah, this was a mistake. It's fixed in the latest commit, where all the fedimg-related tests pass.